webapp/src/app.tsx

```diff
@@ -602,6 +602,11 @@ export class ProjectView
             && this.editorFile && this.editorFile.name == pxt.ASSETS_FILE;
     }
 
+    isJavaScriptEditorActive() {
```

Comment:
I think we need to update the `isPythonActive` function as well. It looks like debug is disabled on custom `.py` files:
![image](https://github.com/user-attachments/assets/d0c23c1b-9c4b-4794-a35c-c56dd8ec620e)

Reply:
Is it possible to put python files in extensions? Should you be able to use those files in the debugger?